tests/h/views/activity_test.py

```diff
@@ -573,10 +575,36 @@ def test_search_passes_the_group_annotation_count_to_the_template(self,
                                                                       factories,
                                                                       test_group,
                                                                       test_user,
+                                                                      query,
+                                                                      annotation_stats_service,
                                                                       pyramid_request):
+        query.extract.return_value = MultiDict({"user": test_user, "group": test_group})
         result = controller.search()['stats']
+        annotation_stats_service.group_annotation_count.assert_called_with(test_group.pubid)
         assert result['annotation_count'] == 5
```

Comment:
It wasn't immediately clear when reading this that we are verifying that the `annotation_count` template value is the same as the return value from `annotation_stats_service.group_annotation_count`.